### PR TITLE
GET subjects and teachers; create teacher-subject connection

### PR DESCRIPTION
**Pull request by andreea-irina**
Updated README file with request/response
Modified login from GET to POST by frontend request
Modified the update in user - was missing getID
GET teachers by subject
GET subjects of classroom - created a new table (classroom_subject) in the database for this
POST teachers to subjects

### CODE REVIEW

File: src/main/java/iss/sirius/Model/Teacher.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
     @Column(name = "lastname")
     private String lastname;
 
-    @OneToOne(cascade = CascadeType.ALL)
+    @OneToOne(cascade = {CascadeType.ALL})
```
Comment (BrigitteIza) [inline src/main/java/iss/sirius/Model/Teacher.java]: de ce ai facut schimbarea asta :o

File: src/test/java/iss/sirius/SiriusApplicationTests.java [hunk 2]
```diff
@@ -6,8 +6,17 @@
 @SpringBootTest
 class SiriusApplicationTests {
 
+    public UserControllerTest userControllerTest;
+
```
Comment (BrigitteIza) [inline src/test/java/iss/sirius/SiriusApplicationTests.java:10]: Merg testele asa?

Comment (andreea-irina) [reply src/test/java/iss/sirius/SiriusApplicationTests.java:10]: Credeam ca o problema reiese de acolo dar nu era si am uitat sa le schimb inapoi :(

Comment (andreea-irina) [reply src/test/java/iss/sirius/SiriusApplicationTests.java:10]: Testele nu merg inca, lucerez la ele in background si vor mai fi modificari. Momentan poti sa le ignori si te anunt eu cand totul e ok

File: src/main/java/iss/sirius/Model/Teacher.java (hunk 1 above)
Comment (andreea-irina) [reply src/main/java/iss/sirius/Model/Teacher.java]: Credeam ca o problema reiese de acolo dar nu era si am uitat sa le schimb inapoi :(

Comment (BrigitteIza) [reply src/main/java/iss/sirius/Model/Teacher.java]: atunci poti sa le faci inapoi? si dupa poti da merge daca zici ca merge :) numai daca ai testat prin postman ca merge

**Pull request merged**